modules/exploits/windows/smb/smb_relay.rb

```diff
@@ -69,678 +74,131 @@ module is not able to clean up after itself. The service and payload
           'StackAdjustment' => -3500
         },
         'References' => [
-          [ 'CVE', '2008-4037'],
-          [ 'OSVDB', '49736'],
-          [ 'MSB', 'MS08-068'],
-          [ 'URL', 'http://blogs.technet.com/swi/archive/2008/11/11/smb-credential-reflection.aspx'],
-          [ 'URL', 'http://en.wikipedia.org/wiki/SMBRelay' ],
-          [ 'URL', 'http://technet.microsoft.com/en-us/sysinternals/bb897553.aspx' ]
+          ['CVE', '2008-4037'],
+          ['OSVDB', '49736'],
+          ['MSB', 'MS08-068'],
+          ['URL', 'http://blogs.technet.com/swi/archive/2008/11/11/smb-credential-reflection.aspx'],
+          ['URL', 'http://en.wikipedia.org/wiki/SMBRelay'],
+          ['URL', 'http://technet.microsoft.com/en-us/sysinternals/bb897553.aspx']
         ],
         'Arch' => [ARCH_X86, ARCH_X64],
         'Platform' => 'win',
         'Targets' => [
-          [ 'Automatic', {} ],
+          ['Automatic', {}],
         ],
         'DisclosureDate' => '2001-03-31',
-        'DefaultTarget' => 0
+        'DefaultTarget' => 0,
+        'Actions' => [['Capture', { 'Description' => 'Run SMB MITM server' }]],
+        'PassiveActions' => ['Capture'],
+        'DefaultAction' => 'Capture'
+        # 'Stance' => Msf::Exploit::Stance::Passive,
+        # 'Passive' => true
       )
     )
 
     register_options(
       [
-        OptAddress.new('SMBHOST', [ false, 'The target SMB server (leave empty for originating system)']),
-        OptString.new('SHARE', [ true, 'The share to connect to', 'ADMIN$' ])
+        # TODO: See if RHosts should be used, for scheme suppport (i.e. HTTP targets) - and renaming this to RELAY_TARGETS?
+        OptAddressRange.new('SMBHOST', [true, 'Target address range or CIDR identifier to relay to']),
```

Comment:
Would probably make sense to make this RHOSTS and then alias SMBHost -> RHOSTS.

Reply:
Will punt on this for now, and add as part of relaying to http targets